[2022-08-06 19:38:15,248] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: our_first_dag_v5.second_task scheduled__2022-03-04T00:00:00+00:00 [queued]>
[2022-08-06 19:38:15,266] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: our_first_dag_v5.second_task scheduled__2022-03-04T00:00:00+00:00 [queued]>
[2022-08-06 19:38:15,266] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2022-08-06 19:38:15,266] {taskinstance.py:1377} INFO - Starting attempt 1 of 6
[2022-08-06 19:38:15,266] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2022-08-06 19:38:15,292] {taskinstance.py:1397} INFO - Executing <Task(BashOperator): second_task> on 2022-03-04 00:00:00+00:00
[2022-08-06 19:38:15,304] {standard_task_runner.py:52} INFO - Started process 2157 to run task
[2022-08-06 19:38:15,307] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'our_first_dag_v5', 'second_task', 'scheduled__2022-03-04T00:00:00+00:00', '--job-id', '655', '--raw', '--subdir', 'DAGS_FOLDER/pipeline.py', '--cfg-path', '/tmp/tmp59amr45t', '--error-file', '/tmp/tmpv6n51kex']
[2022-08-06 19:38:15,307] {standard_task_runner.py:80} INFO - Job 655: Subtask second_task
[2022-08-06 19:38:15,412] {task_command.py:371} INFO - Running <TaskInstance: our_first_dag_v5.second_task scheduled__2022-03-04T00:00:00+00:00 [running]> on host b29c168c1666
[2022-08-06 19:38:15,531] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=coder2j
AIRFLOW_CTX_DAG_ID=our_first_dag_v5
AIRFLOW_CTX_TASK_ID=second_task
AIRFLOW_CTX_EXECUTION_DATE=2022-03-04T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-03-04T00:00:00+00:00
[2022-08-06 19:38:15,532] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-08-06 19:38:15,532] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'echo hey, I am task2 and will be running after task1!']
[2022-08-06 19:38:15,546] {subprocess.py:85} INFO - Output:
[2022-08-06 19:38:15,548] {subprocess.py:92} INFO - hey, I am task2 and will be running after task1!
[2022-08-06 19:38:15,549] {subprocess.py:96} INFO - Command exited with return code 0
[2022-08-06 19:38:15,594] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=our_first_dag_v5, task_id=second_task, execution_date=20220304T000000, start_date=20220806T193815, end_date=20220806T193815
[2022-08-06 19:38:15,645] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-08-06 19:38:15,691] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2022-08-06 21:52:06,492] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: our_first_dag_v5.second_task scheduled__2022-03-04T00:00:00+00:00 [queued]>
[2022-08-06 21:52:06,504] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: our_first_dag_v5.second_task scheduled__2022-03-04T00:00:00+00:00 [queued]>
[2022-08-06 21:52:06,504] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2022-08-06 21:52:06,504] {taskinstance.py:1377} INFO - Starting attempt 1 of 6
[2022-08-06 21:52:06,504] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2022-08-06 21:52:06,519] {taskinstance.py:1397} INFO - Executing <Task(BashOperator): second_task> on 2022-03-04 00:00:00+00:00
[2022-08-06 21:52:06,525] {standard_task_runner.py:52} INFO - Started process 2338 to run task
[2022-08-06 21:52:06,528] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'our_first_dag_v5', 'second_task', 'scheduled__2022-03-04T00:00:00+00:00', '--job-id', '656', '--raw', '--subdir', 'DAGS_FOLDER/pipeline.py', '--cfg-path', '/tmp/tmpolh0zd71', '--error-file', '/tmp/tmp22hu9zd5']
[2022-08-06 21:52:06,528] {standard_task_runner.py:80} INFO - Job 656: Subtask second_task
[2022-08-06 21:52:06,592] {task_command.py:371} INFO - Running <TaskInstance: our_first_dag_v5.second_task scheduled__2022-03-04T00:00:00+00:00 [running]> on host c3f731879890
[2022-08-06 21:52:06,666] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=coder2j
AIRFLOW_CTX_DAG_ID=our_first_dag_v5
AIRFLOW_CTX_TASK_ID=second_task
AIRFLOW_CTX_EXECUTION_DATE=2022-03-04T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-03-04T00:00:00+00:00
[2022-08-06 21:52:06,667] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-08-06 21:52:06,667] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'echo hey, I am task2 and will be running after task1!']
[2022-08-06 21:52:06,680] {subprocess.py:85} INFO - Output:
[2022-08-06 21:52:06,681] {subprocess.py:92} INFO - hey, I am task2 and will be running after task1!
[2022-08-06 21:52:06,682] {subprocess.py:96} INFO - Command exited with return code 0
[2022-08-06 21:52:06,710] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=our_first_dag_v5, task_id=second_task, execution_date=20220304T000000, start_date=20220806T215206, end_date=20220806T215206
[2022-08-06 21:52:06,739] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-08-06 21:52:06,768] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2022-08-06 21:57:30,435] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: our_first_dag_v5.second_task scheduled__2022-03-04T00:00:00+00:00 [queued]>
[2022-08-06 21:57:30,446] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: our_first_dag_v5.second_task scheduled__2022-03-04T00:00:00+00:00 [queued]>
[2022-08-06 21:57:30,447] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2022-08-06 21:57:30,447] {taskinstance.py:1377} INFO - Starting attempt 1 of 6
[2022-08-06 21:57:30,447] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2022-08-06 21:57:30,461] {taskinstance.py:1397} INFO - Executing <Task(BashOperator): second_task> on 2022-03-04 00:00:00+00:00
[2022-08-06 21:57:30,468] {standard_task_runner.py:52} INFO - Started process 2131 to run task
[2022-08-06 21:57:30,470] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'our_first_dag_v5', 'second_task', 'scheduled__2022-03-04T00:00:00+00:00', '--job-id', '656', '--raw', '--subdir', 'DAGS_FOLDER/pipeline.py', '--cfg-path', '/tmp/tmpye4hiq4u', '--error-file', '/tmp/tmpl262m_uf']
[2022-08-06 21:57:30,471] {standard_task_runner.py:80} INFO - Job 656: Subtask second_task
[2022-08-06 21:57:30,550] {task_command.py:371} INFO - Running <TaskInstance: our_first_dag_v5.second_task scheduled__2022-03-04T00:00:00+00:00 [running]> on host bc7e154a4fdd
[2022-08-06 21:57:30,639] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=coder2j
AIRFLOW_CTX_DAG_ID=our_first_dag_v5
AIRFLOW_CTX_TASK_ID=second_task
AIRFLOW_CTX_EXECUTION_DATE=2022-03-04T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-03-04T00:00:00+00:00
[2022-08-06 21:57:30,640] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-08-06 21:57:30,641] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'echo hey, I am task2 and will be running after task1!']
[2022-08-06 21:57:30,655] {subprocess.py:85} INFO - Output:
[2022-08-06 21:57:30,656] {subprocess.py:92} INFO - hey, I am task2 and will be running after task1!
[2022-08-06 21:57:30,656] {subprocess.py:96} INFO - Command exited with return code 0
[2022-08-06 21:57:30,684] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=our_first_dag_v5, task_id=second_task, execution_date=20220304T000000, start_date=20220806T215730, end_date=20220806T215730
[2022-08-06 21:57:30,722] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-08-06 21:57:30,751] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2022-08-06 22:12:33,243] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: our_first_dag_v5.second_task scheduled__2022-03-04T00:00:00+00:00 [queued]>
[2022-08-06 22:12:33,260] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: our_first_dag_v5.second_task scheduled__2022-03-04T00:00:00+00:00 [queued]>
[2022-08-06 22:12:33,260] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2022-08-06 22:12:33,261] {taskinstance.py:1377} INFO - Starting attempt 1 of 6
[2022-08-06 22:12:33,261] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2022-08-06 22:12:33,282] {taskinstance.py:1397} INFO - Executing <Task(BashOperator): second_task> on 2022-03-04 00:00:00+00:00
[2022-08-06 22:12:33,320] {standard_task_runner.py:52} INFO - Started process 2152 to run task
[2022-08-06 22:12:33,326] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'our_first_dag_v5', 'second_task', 'scheduled__2022-03-04T00:00:00+00:00', '--job-id', '659', '--raw', '--subdir', 'DAGS_FOLDER/pipeline.py', '--cfg-path', '/tmp/tmp0aqdf403', '--error-file', '/tmp/tmpqq8db2ch']
[2022-08-06 22:12:33,327] {standard_task_runner.py:80} INFO - Job 659: Subtask second_task
[2022-08-06 22:12:33,417] {task_command.py:371} INFO - Running <TaskInstance: our_first_dag_v5.second_task scheduled__2022-03-04T00:00:00+00:00 [running]> on host 1250c3b659cd
[2022-08-06 22:12:33,513] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=coder2j
AIRFLOW_CTX_DAG_ID=our_first_dag_v5
AIRFLOW_CTX_TASK_ID=second_task
AIRFLOW_CTX_EXECUTION_DATE=2022-03-04T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-03-04T00:00:00+00:00
[2022-08-06 22:12:33,514] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-08-06 22:12:33,515] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'echo hey, I am task2 and will be running after task1!']
[2022-08-06 22:12:33,528] {subprocess.py:85} INFO - Output:
[2022-08-06 22:12:33,529] {subprocess.py:92} INFO - hey, I am task2 and will be running after task1!
[2022-08-06 22:12:33,530] {subprocess.py:96} INFO - Command exited with return code 0
[2022-08-06 22:12:33,563] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=our_first_dag_v5, task_id=second_task, execution_date=20220304T000000, start_date=20220806T221233, end_date=20220806T221233
[2022-08-06 22:12:33,620] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-08-06 22:12:33,661] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2022-08-06 22:23:56,037] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: our_first_dag_v5.second_task scheduled__2022-03-04T00:00:00+00:00 [queued]>
[2022-08-06 22:23:56,066] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: our_first_dag_v5.second_task scheduled__2022-03-04T00:00:00+00:00 [queued]>
[2022-08-06 22:23:56,066] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2022-08-06 22:23:56,066] {taskinstance.py:1377} INFO - Starting attempt 1 of 6
[2022-08-06 22:23:56,066] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2022-08-06 22:23:56,091] {taskinstance.py:1397} INFO - Executing <Task(BashOperator): second_task> on 2022-03-04 00:00:00+00:00
[2022-08-06 22:23:56,099] {standard_task_runner.py:52} INFO - Started process 2218 to run task
[2022-08-06 22:23:56,103] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'our_first_dag_v5', 'second_task', 'scheduled__2022-03-04T00:00:00+00:00', '--job-id', '662', '--raw', '--subdir', 'DAGS_FOLDER/pipeline.py', '--cfg-path', '/tmp/tmpf9iedpac', '--error-file', '/tmp/tmp97il0pew']
[2022-08-06 22:23:56,103] {standard_task_runner.py:80} INFO - Job 662: Subtask second_task
[2022-08-06 22:23:56,189] {task_command.py:371} INFO - Running <TaskInstance: our_first_dag_v5.second_task scheduled__2022-03-04T00:00:00+00:00 [running]> on host b215076695c3
[2022-08-06 22:23:56,284] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=coder2j
AIRFLOW_CTX_DAG_ID=our_first_dag_v5
AIRFLOW_CTX_TASK_ID=second_task
AIRFLOW_CTX_EXECUTION_DATE=2022-03-04T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-03-04T00:00:00+00:00
[2022-08-06 22:23:56,285] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-08-06 22:23:56,285] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'echo hey, I am task2 and will be running after task1!']
[2022-08-06 22:23:56,301] {subprocess.py:85} INFO - Output:
[2022-08-06 22:23:56,303] {subprocess.py:92} INFO - hey, I am task2 and will be running after task1!
[2022-08-06 22:23:56,303] {subprocess.py:96} INFO - Command exited with return code 0
[2022-08-06 22:23:56,338] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=our_first_dag_v5, task_id=second_task, execution_date=20220304T000000, start_date=20220806T222356, end_date=20220806T222356
[2022-08-06 22:23:56,395] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-08-06 22:23:56,443] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
